src/controllers/admin.js

```diff
@@ -1,14 +1,18 @@
 const { insertArticle } = require('../model/queries/addData');
 
 exports.get = (req, res) => {
-  console.log(res.locals.unlockCookie);
-  if (res.locals.unlockCookie.permission === 'admin') {
-    res.render('admin', {
-      title: 'Admin Panel', headerFound: true, footerFound: true, asideFound: true, style: ['admin', 'header', 'footer'], javascript: ['hamburger', 'admin'], login: (res.locals.unlockCookie === null), username: (res.locals.unlockCookie === null) ? 'Unkown' : res.locals.unlockCookie.username,
-    });
+  if (res.locals.unlockCookie) {
+    if (res.locals.unlockCookie.permission === 'admin') {
+      res.render('admin', {
+        title: 'Admin Panel', headerFound: true, footerFound: true, asideFound: true, style: ['admin', 'header', 'footer'], javascript: ['hamburger', 'admin'], login: (res.locals.unlockCookie === null), username: (res.locals.unlockCookie === null) ? 'Unkown' : res.locals.unlockCookie.username,
+      });
+    } else {
+      res.clearCookie('jwt');
```

Comment:
you don't  have to clear the cookie if he is not admin all you need is to prevent him from accessing this page by redirecting him to other page